org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/service/ServiceUseExtension.java

```diff
@@ -52,8 +52,22 @@
 		private long			timeout		= TrackServices.DEFAULT_TIMEOUT;
 
 		/**
+		 * Create with default BundleContextExtension.
+		 *
 		 * @param serviceType of the service
 		 */
+		public Builder(Class<T> serviceType) {
+			this.serviceType = serviceType;
```

Comment:
Isn't this just `this(serviceType, new BundleContextExtension());` So it reuses the other constructor?

Reply:
sure, fixing